mmf/trainers/core/training_loop.py

```diff
@@ -10,7 +10,8 @@
 from mmf.common.report import Report
 from mmf.common.sample import to_device
 from mmf.utils.distributed import is_xla
-from mmf.utils.general import clip_gradients, get_max_updates
+from mmf.utils.flow import extract_loss, get_max_updates
```

Comment:
Since now there are two places who are going to use those functions, I think it is much cleaner to move them to a utils file where both can access. I thought calling it flow makes sense because it could be used for both training and evaluation work flow.